gax-java/gax/src/main/java/com/google/api/gax/batching/ThresholdBatcher.java

```diff
@@ -100,11 +102,21 @@ private ThresholdBatcher(Builder<E> builder) {
     resetThresholds();
   }
 
+  @VisibleForTesting
+  org.threeten.bp.Duration getMaxDelay() {
+    return toThreetenDuration(this.maxDelay);
+  }
```

Comment:
Do we need to introduce this new threetenbp method since it's package-private and didn't exist before? Is it possible to test without adding this in?

Reply:
Modified the test to use captures instead of adding a test-only method